docs_src/rtsf_at_checkout_events/checkout_events.md

```diff
@@ -96,7 +96,7 @@ Example event:
 Scale events track items on the scale. There is only one scale event type required for this reference design, which is:
 
 #### Scale Item 
-`scale-item` occurs when an item has been placed or removed from the security scale.
+`weight` occurs when an item has been placed or removed from the security scale.
```

Comment:
in this context, should we still use scale item and explain the item or value is actually `weight`?

Reply:
i think i will rename `Scale Item` to 'Weight'

Reply:
but the sentence seems to read a bit awkward if using Weight.

Reply:
Scale Weight Reading